projects/fourier/lib/fourier/services/test/tuist/unit.rb

```diff
@@ -6,6 +6,9 @@ module Tuist
         class Unit < Base
           def call
             Utilities::System.tuist("test")
+            Dir.chdir(Constants::TUIST_DIRECTORY) do
+              Utilities::System.system("swift", "test")
```

Comment:
@pepibumur could you afterwards look at this if it makes sense? I was not completely sure about this one. But `swift run tuist test` was not running all the tests.

Reply:
Right. This is correct.